.github/workflows/CI.yml

```diff
@@ -31,12 +31,17 @@ jobs:
           echo "/home/runner/.local/share/solana/install/active_release/bin" >> $GITHUB_PATH
 
       - name: Build test programs
-        run: cd test_programs && cargo build-sbf
+        run: cd svm/test_programs && cargo build-sbf
 
       - name: Run tests
         run: cargo test
         env:
           RUST_BACKTRACE: 1
+      
+      - name: Run SPL tests
+        run: cargo test --features token-2022
```

Comment:
let's add another test run without the token-2022 feature too

Reply:
The name is not perfectly correct but normally it runs two times. One without token2022 and one with it